packages/ERTP/test/unitTests/mintObj.test.js

```diff
@@ -110,11 +114,14 @@ test('mint.mintPayment set AssetKind with invites', async t => {
 // This test models ballet tickets
 test('non-fungible tokens example', async t => {
   t.plan(11);
+
   const {
     mint: balletTicketMint,
     issuer: balletTicketIssuer,
     brand,
-  } = makeIssuerKit('Agoric Ballet Opera tickets', AssetKind.SET);
+  } = /**
+   * @type {IssuerKit<'set', { seat: number; show: string; start: string }>}
+   */ (makeIssuerKit('Agoric Ballet Opera tickets', AssetKind.SET));
```

Comment:
Does it make sense to also pass a corresponding `elementShape` to this call to `makeIssuerKit`?

As a test, maybe not worth bothering, so I ask hypothetically: would it have made sense, if code like this appears in production?